mono/metadata/icall.c

```diff
@@ -3646,40 +3642,38 @@ ves_icall_System_Enum_get_hashcode (MonoObject *eobj)
 }
 
 ICALL_EXPORT MonoBoolean
-ves_icall_System_Enum_GetEnumValuesAndNames (MonoReflectionType *type, MonoArray **values, MonoArray **names)
+ves_icall_System_Enum_GetEnumValuesAndNames (MonoReflectionTypeHandle type, MonoArrayHandle values, MonoArrayHandle names, MonoError *error)
```

Comment:
I took a while to find out that this was the icall that uses outargs.

We need a marker on the runtime size to tell that those handles are outarg.